config/config-loader.js

```diff
@@ -0,0 +1,24 @@
+/** Node module that will be executed at build-time */
+
+const convict = require("convict");
+const dotenv = require("dotenv");
+
+// Reads in the environment variables and saves it into the process.env global node variable
+dotenv.config();
+
+// Create a configuration schema
+const config = convict({
+  grafana_url: {
+    default: "",
```

Comment:
there's no default although the README says htere is

Reply:
Oh, yeah you're totally right, I'll update the README